microservice/src/main/resources/application.yml

```diff
@@ -0,0 +1,35 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:5432/skt-task
+    username: jxjxjxm
```

Comment:
I recommend to use here same approach than for username and password of RabbitMQ (with `${RABBIT_USERNAME}`)